File: app/src/main/java/com/kushkipagos/android/KushkiEnvironment.java
package com.kushkipagos.android;

public enum KushkiEnvironment {
    LOCAL("http://localhost:8888/kushki/api/v1"),
    TESTING("https://uat.aurusinc.com/kushki/api/v1"),
    STAGING("https://staging.aurusinc.com/kushki/api/v1"),
    PRODUCTION("https://p1.kushkipagos.com/kushki/api/v1");

    private String url;

    KushkiEnvironment(String url) {
        this.url = url;
    }

    public String getUrl() {
        return url;
    }
}

File: kushki/src/main/java/com/kushkipagos/kushki/KushkiEnvironment.kt
package com.kushkipagos.kushki

enum class KushkiEnvironment(override val url: String) : Environment {
    TESTING("https://api-uat.kushkipagos.com/"),
    QA("https://api-qa.kushkipagos.blue/"),
    CI("https://api-ci.kushkipagos.com/"),
    STAGING("https://api-stg.kushkipagos.com/"),
    PRODUCTION("https://api.kushkipagos.com/"),
    UAT_REGIONAL("https://regional-uat.kushkipagos.com/"),
    PRODUCTION_REGIONAL("https://regional.kushkipagos.com/");

}
